packages/medusa-plugin-algolia/src/services/algolia.ts

```diff
@@ -56,11 +64,20 @@ class AlgoliaService extends SearchService {
    * @param {*} type
    * @return {*}
    */
-  async addDocuments(indexName, documents, type) {
-    const transformedDocuments = this.getTransformedDocuments(type, documents)
-    return await this.client_
-      .initIndex(indexName)
-      .saveObjects(transformedDocuments)
+  async addDocuments(indexName: string, documents: any, type: string) {
+    const transformedDocuments = await this.getTransformedDocuments(
+      type,
+      documents
+    )
+
+    try {
+      return await this.client_
+        .initIndex(indexName)
+        .saveObjects(transformedDocuments)
+    } catch (error) {
+      console.log(error)
```

Comment:
todo: we should use the logger instead of console

Reply:
Did not mean to have the log in there. Will remove entirely